### PR TITLE
Docs: Add link to llm-setup page in cloud ML docs

### PR DESCRIPTION
**Pull request by edwardcqian**

### CODE REVIEW

File: packages/grafana-llm-app/src/README.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Grafana Cloud: the LLM app plugin is installed for everyone, but LLM features ar
 
 OSS or Enterprise: install and configure this plugin with your OpenAI-compatible API key to enable LLM-powered features across Grafana.
 
+For more detailed setup instructions see [LLM plugin](https://grafana.com/docs/grafana-cloud/alerting-and-irm/machine-learning/llm/llm-setup/) section under machine learning.
```
Comment (JohnnyK-Grafana) [inline packages/grafana-llm-app/src/README.md:27]:
```suggestion
For more detailed setup instructions see [LLM plugin](/docs/grafana-cloud/alerting-and-irm/machine-learning/llm/llm-setup/) section under machine learning.
```

Comment (JohnnyK-Grafana) [reply packages/grafana-llm-app/src/README.md:27]: one tweak, otherwise lgtm!

**Comment by obetomuniz** [comment]
Just consider Johnny nit. Otherwise, lgtm! 😀

File: packages/grafana-llm-app/src/README.md (hunk 1 above)
Comment (edwardcqian) [reply packages/grafana-llm-app/src/README.md:27]: This linked to the cloud machine-learning docs and not docs in this repo, I think we need the grafana url header for this.

**Pull request merged**